<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choix de fichiers à supprimer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/scolasync.svg</normaloff>images/scolasync.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Zone de recherche</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Répertoire des &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;documents de &lt;br /&gt;travail&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="travailEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Le répertoire des clés&lt;br&gt;où se trouvent les&lt;br&gt;documents de travail</string>
           </property>
           <property name="statusTip">
            <string>Le répertoire des clés où se trouvent les documents de travail</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Choix de clé modèle</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListView" name="listChoixCle">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Choisir une des clés</string>
           </property>
           <property name="statusTip">
            <string>Choisir une des clés</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>200</height>
         </size>
        </property>
        <property name="title">
         <string>Liste de fichiers à traiter (jokers autorisés)</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QListView" name="listView"/>
         </item>
         <item>
          <widget class="QFrame" name="frame">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QWidget" name="widget_2" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QWidget" name="widget_3" native="true">
                 <layout class="QHBoxLayout" name="horizontalLayout">
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QToolButton" name="minusButton">
                    <property name="toolTip">
                     <string>Supprimer l'item sélectionné</string>
                    </property>
                    <property name="statusTip">
                     <string>Supprimer l'item sélectionné</string>
                    </property>
                    <property name="text">
                     <string>...</string>
                    </property>
                    <property name="icon">
                     <iconset>
                      <normaloff>../../../../../../usr/share/icons/Tango/scalable/actions/remove.svg</normaloff>../../../../../../usr/share/icons/Tango/scalable/actions/remove.svg</iconset>
                    </property>
                    <property name="iconSize">
                     <size>
                      <width>24</width>
                      <height>24</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="plusButton">
                    <property name="toolTip">
                     <string>Ajouter un fichier ou un filtre défini dans la ligne d'édition</string>
                    </property>
                    <property name="statusTip">
                     <string>Ajouter un fichier ou un filtre défini dans la ligne d'édition</string>
                    </property>
                    <property name="text">
                     <string>...</string>
                    </property>
                    <property name="icon">
                     <iconset>
                      <normaloff>../../../../../../usr/share/icons/Tango/scalable/actions/add.svg</normaloff>../../../../../../usr/share/icons/Tango/scalable/actions/add.svg</iconset>
                    </property>
                    <property name="iconSize">
                     <size>
                      <width>24</width>
                      <height>24</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit">
                 <property name="toolTip">
                  <string>Ligne d'édition du nom d'un fichier ou d'un répertoire.&lt;br&gt;Jokers autorisés</string>
                 </property>
                 <property name="statusTip">
                  <string>Ligne d'édition du nom d'un fichier ou d'un répertoire.&lt;br&gt;Jokers autorisés</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QToolButton" name="chooseButton">
                 <property name="toolTip">
                  <string>Rechercher (fichier) ...</string>
                 </property>
                 <property name="statusTip">
                  <string>Rechercher (fichier) ...</string>
                 </property>
                 <property name="text">
                  <string>...</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../../../../../../usr/share/icons/Tango/scalable/actions/fileopen.svg</normaloff>../../../../../../usr/share/icons/Tango/scalable/actions/fileopen.svg</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>48</width>
                   <height>40</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="chooseButton_dir">
                 <property name="toolTip">
                  <string>Rechercher (répertoire) ...</string>
                 </property>
                 <property name="statusTip">
                  <string>Rechercher (répertoire) ...</string>
                 </property>
                 <property name="text">
                  <string>...</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../../../../../../usr/share/icons/Tango/scalable/places/folder.svg</normaloff>../../../../../../usr/share/icons/Tango/scalable/places/folder.svg</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>48</width>
                   <height>40</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
